bindings/nodejs/index.js

```diff
@@ -19,9 +19,89 @@
 
 /// <reference types="node" />
 
+const { Writable, Readable } = require('node:stream')
+
+class ReadStream extends Readable {
```

Comment:
I feel like we need `BlockingReadStream` and `ReadStream` for `BlockingReader` and `Reader`.

Reply:
I'm adding this.